artemis/modules/wordpress_plugins.py

```diff
@@ -244,13 +243,9 @@ def _get_plugins_from_homepage(self, url: str) -> List[Dict[str, Any]]:
             match = re.search(pattern, link)
             if match:
                 slug = match.group(1)
-                data = json.loads(
-                    self.cached_get(
-                        f"https://api.wordpress.org/plugins/info/1.0/{slug}.json",
-                        "version-" + slug,
-                        timeout=3600,
-                    )
-                )
+                data = FallbackAPICache.get(
```

Comment:
but then what's the point of having predefined URLs in class fields when you access other with defined url

Reply:
because the predefined ones will be warmed up so in case the first request after starting the container doesn't work, a stale response will be used